index.js

```diff
@@ -34,6 +34,32 @@ Retryme.prototype.attempt = function (action, fn) {
   }));
 };
 
+Retryme.prototype.async = function (asyncfn) {
+  return new Promise((fulfill, reject) => {
```

Comment:
Is it okay to force Promises on consumers? Does this need to support client-side usage where Promise may not exist?